src/Storefront/Resources/views/storefront/page/account/profile/recover-password.html.twig

```diff
@@ -57,7 +57,8 @@
                                             {% block page_account_profile_recover_password_action_send %}
                                                 <button type="submit"
                                                         class="account-recover-password-submit btn btn-primary"
-                                                        title="{{ 'account.profileRecoverPasswordSubmit'|trans }}">
+                                                        title="{{ 'account.profileRecoverPasswordSubmit'|trans }}"
+                                                        onclick="this.disabled = true; this.closest('form').submit();">
```

Comment:
Hey @sneakyvv,

good catch 👍 I can definitely reproduce the issue and receive multiple emails when spamming the button.

I think we don't need an extra inline script to prevent this. There is already a `FormSubmitLoaderPlugin` which is also used in other areas (e.g. checkout confirm button) to prevent this kind of issue. It also adds a loading indicator to the submit button of the form when loading.

It should work already by adding attribute `data-form-submit-loader="true"` to the form `account-recover-password-form`. I tested it briefly and for me it solves the issue.

Reply:
Good to know!
It was indeed a quick fix, but probably not the best, so I was expecting some feedback 😄 
I'll adjust the PR accordingly 👍